opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -59,18 +62,25 @@ pub trait LogProcessor: Send + Sync + Debug {
 #[derive(Debug)]
 pub struct SimpleLogProcessor {
     exporter: Mutex<Box<dyn LogExporter>>,
+    is_shutdown: AtomicBool,
 }
 
 impl SimpleLogProcessor {
     pub(crate) fn new(exporter: Box<dyn LogExporter>) -> Self {
         SimpleLogProcessor {
             exporter: Mutex::new(exporter),
+            is_shutdown: AtomicBool::new(false),
         }
     }
 }
 
 impl LogProcessor for SimpleLogProcessor {
     fn emit(&self, data: LogData) {
+        // noop after shutdown
+        if self.is_shutdown.load(std::sync::atomic::Ordering::Relaxed) {
+            return;
+        }
```

Comment:
Do we need this check, or else rely on the exporter to check this before actual export. As  I don't think batchprocessor is doing this check. Just want to ensure both the processors are consistent here :)
Also, good to compare the [benchmark](https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry-sdk/benches/log.rs) if this validation has any effect on the performance.

Reply:
> As I don't think batchprocessor is doing this check. 

After batch processor shuts down the channel used to send logs to background task will error out because the recevier has been dropped. So we get the noop(well, in this case it will emit an error) free